ethereum_history_api/oracles/src/noir/oracles/common/encode.test.ts

```diff
@@ -50,6 +58,21 @@ describe('encodeField', () => {
   });
 });
 
+describe('encodeU128', () => {
+  it('should encode 1 correctly', () => {
+    expect(encodeU128(1n)).toStrictEqual(['0x', '0x01']);
+  });
+  it('should encode 255 correctly', () => {
+    expect(encodeU128(255n)).toStrictEqual(['0x', '0xff']);
+  });
+  it('should encode large values correctly', () => {
+    expect(encodeU128(100500n)).toStrictEqual(['0x', '0x018894']);
+  });
+  it('should encode values over 64 bits correctly', () => {
+    expect(encodeU128(2n ** 64n)).toStrictEqual(['0x01', '0x01']);
```

Comment:
2**64 should be encoded to ['0x01', '0x00'], this test is wrong and there must be some bug in implementation